src/components/Input/index.tsx

```diff
@@ -0,0 +1,17 @@
+import TextArea from './TextArea/TextArea';
+import { InputProps } from './InputProps';
+import InternalInput from './Input';
+
+export { InputProps } from './InputProps';
+export { TextAreaProps } from './TextArea/textAreaProps';
+
+interface CompoundedComponent
```

Comment:
could have a better name for this interface

how about InputComponents or InputCompoundedComponents. wdyt? 

but 3 word variable name is not recommended unless it is needed to explain the variable.